windows_containers/understanding-windows-container-workloads.adoc

```diff
@@ -26,6 +26,11 @@ Multi-tenancy for clusters that have Windows nodes is not supported. Hostile mul
 Windows Server Containers provide resource isolation using a shared kernel but are not intended to be used in hostile multitenancy scenarios. Scenarios that involve hostile multitenancy should use Hyper-V Isolated Containers to strongly isolate tenants.
 ====
 
+include::modules/wmco-prerequisites.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
So, grasshopper, you've been introduced into the ways of subverting our limitations for xrefs.  Well done.